themes/bakke/README.md

```diff
@@ -0,0 +1,15 @@
+# Bakke-Dark
+
+![alt text](bakke-dark.png)
+
+- **Version Control State Indicators:** It uses brown to indicate a dirty state and bold green for a clean state in version control-related prompts such as Git.
+
+- **Prompt Prefix and Suffix:** It uses green as a prefix and suffix in version control-related prompts.
+
+- **Virtual Environments:** It displays information about Python virtual environments, such as virtualenv and condaenv, in parentheses and with specific colors.
+
+- **Main Prompt Structure:** It uses bold gray to form the main structure of the prompt, which includes the username, host, and current working directory.
+
+- **Additional Features:** It allows displaying additional information, such as the Ruby environment if present, and the use of RVM.
+
+`Overall, this theme offers a pleasant and functional user experience when working in the terminal, with a focus on clarity and relevant information for the user.`
```

Comment:
<code>\`...\`</code> is used to include the codes, which need to be verbatim. Here "Overall, ... user." is normal text, so should be presented in a plain manner.